frontend/src/pages/new_document.tsx

```diff
@@ -220,108 +274,122 @@ export function NewDocumentPage() {
                 <p className="text-red-600 text-sm mt-0.5">Audio file is required.</p>
               )}
             </div>
-            {!isLoading ? (
-              <div className="flex row">
-                <FormControl label="Model" error={errors.model?.message} className="flex-grow mr-2">
-                  <Select {...register('model')}>
-                    {Object.values(config.models).map((cur_model) =>
-                      cur_model !== undefined ? (
-                        <option value={cur_model.id} key={cur_model.id}>
-                          {cur_model.name}
-                        </option>
-                      ) : (
-                        <></>
-                      ),
-                    )}
-                  </Select>
-                </FormControl>
-                <FormControl
-                  label="Language"
-                  error={errors.language?.message}
-                  className="flex-grow"
-                >
-                  <Select {...register('language')}>
-                    {getLanguages(config.models, model)?.map((lang) => (
-                      <option value={lang} key={lang}>
-                        {lang}
-                      </option>
-                    ))}
-                  </Select>
-                </FormControl>
-              </div>
-            ) : (
-              <></>
-            )}
+            {!isImport && (
```

Comment:
Can I have some alternative text when we are importing?

Reply:
What do you mean?

Reply:
I would like to have some kind of feedback acknoledging that we are now importing an existing document. Also I would like to have some indicatino that this is possible here.